dspace/config/spring/api/general-import-service.xml

```diff
@@ -22,7 +22,7 @@
           lazy-init="false" autowire="byType" destroy-method="destroy"/>
 
     <bean class="java.lang.String" id="apiKey" autowire-candidate="true">
-        <constructor-arg value="${external-sources.elsevier.key}"/>
+        <constructor-arg value=""/>
```

Comment:
Why is this now an empty argument? That seems rather odd to me, as it implies to me that to enable this, a user will need to edit this Spring config?  I'd recommend against forcing users to modify Spring configs, if at all possible.